build/Dockerfile

```diff
@@ -3,10 +3,11 @@
 # Run the following commands from the root folder to build, run and kill the application
 # >> docker build -f build/Dockerfile -t ndb-admin:latest .
 # >> docker run -p=80:80 ndb-admin:latest
-FROM node:16.13.0-alpine3.13 as builder
+FROM node:20-alpine as builder
```

Comment:
should we fix this to a more specific minor version? In theory, just major version like this should be enough, I guess?